bundles/org.openhab.binding.rotel/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,178 @@
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="recordSource" typeId="recordSource" />
+			<channel id="dsp" typeId="dsp" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+			<channel id="bass" typeId="bass" />
+			<channel id="treble" typeId="treble" />
+			<channel id="line1" typeId="frontPanelLine" />
+			<channel id="line2" typeId="frontPanelLine" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="mainZoneType2">
+		<label>Main Zone</label>
+		<description>The controls of the main zone</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="recordSource" typeId="recordSource" />
+			<channel id="dsp" typeId="dsp" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+			<channel id="line1" typeId="frontPanelLine" />
+			<channel id="line2" typeId="frontPanelLine" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="mainZoneType3">
+		<label>Main Zone</label>
+		<description>The controls of the main zone</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="recordSource" typeId="recordSource" />
+			<channel id="dsp" typeId="dsp" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+			<channel id="bass" typeId="bass" />
+			<channel id="treble" typeId="treble" />
+			<channel id="line1" typeId="frontPanelLine" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone2type1">
+		<label>Zone 2</label>
+		<description>The controls of the zone 2</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone2type2">
+		<label>Zone 2</label>
+		<description>The controls of the zone 2</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone3">
+		<label>Zone 3</label>
+		<description>The controls of the zone 3</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone4">
+		<label>Zone 4</label>
+		<description>The controls of the zone 4</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="power">
```

Comment:
No need for this channel type, `system.power` should do.

Reply:
Done.
I kept the specific channel type (renamed into zonePower) for the additional zones.

Reply:
Why? You also had used a single channel type before.

Reply:
See my answer in the other review comment.